packages/ncids-css/scss/components/usa-footer/internal/_nci-big.scss

```diff
@@ -1,50 +1,69 @@
 .usa-footer--nci-big {
```

Comment:
This will be merged in a separate ticket. Note - footer Icon styles in  `docs/src/index.scss` should be removed, all styles are copied to this partial.